src/Func.h

```diff
@@ -216,13 +216,60 @@ class Stage {
     EXPORT Stage &gpu(VarOrRVar block_x, VarOrRVar block_y, VarOrRVar block_z,
                       VarOrRVar thread_x, VarOrRVar thread_y, VarOrRVar thread_z,
                       DeviceAPI device_api = DeviceAPI::Default_GPU);
+
+    // TODO(psuriana): For now we need to expand "tx" into Var and RVar versions
```

Comment:
The reason why I explicitly declare `gpu_tile(VarOrRVar x, VarOrRVar bx, Var tx...` or `gpu_tile(VarOrRVar x, VarOrRVar bx, RVar tx...` is because Var can be implicitly converted into either VarOrRVar or Expr which confuses the compiler. They conflict with this old GPU interface: 
```
EXPORT Stage &gpu_tile(VarOrRVar x, VarOrRVar y,
                           Expr x_size, Expr y_size,
                           TailStrategy tail = TailStrategy::Auto,
                           DeviceAPI device_api = DeviceAPI::Default_GPU);
```